crypto/digest_extra/digest_test.cc

```diff
@@ -176,10 +190,10 @@ static void CompareDigest(const DigestTestVector *test,
 }
 
 static void TestDigest(const DigestTestVector *test) {
-    // Test SHA3 by enabling |unstable_sha3_enabled_flag|, then disable it
-    // |unstable_sha3_enabled_flag| is desabled by default
-    // SHA3 negative tests are implemented in /fipsmodule/sha/sha3_test.cc
-    if ((strcmp(test->md.name, "SHA3-256") == 0)) {
+    // Test SHA3 only when |unstable_enable_sha3| is enabled
```

Comment:
IIRC we changed this wording to the existing one? Hence, the wording should stay the same, but we should add the extra if conditional.